### PR TITLE
[FEAT] - Onboarding

### PR DESCRIPTION
**Pull request by RemiBonnet**
Hey @gromon42 @bdebon @WilliamTraoreee 👋,

I added the `personalize` and `company` steps with the right behavior and the API connexion.
And, I added a form behavior with this library: [React Hook Form](https://react-hook-form.com/) with the manage of errors, default values, validators, etc.

Feel free to send me feedback, if you want, so I can quickly update

### CODE REVIEW

**Review by bdebon — COMMENTED** [review]
Hey @RemiBonnet,
I like the behavior of the react form library you choosed. It's easy to use and I have the sensation that it will answer all our usecase.
I made some suggestions here and there.
Great job overall. Nice to see some real field work in this v3 architecture!

File: libs/pages/onboarding/feature/src/lib/components/form-company/form-company.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    handleSubmit,
+    control,
+    setValue,
+    getValues,
+    formState: { errors },
+  } = useForm()
+
+  useEffect(() => {
+    if (userSignUp) {
+      setValue('company_name', userSignUp?.company_name || undefined)
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/pages/onboarding/feature/src/lib/components/form-company/form-company.tsx]: Yep, you are right, it's useless in this case, fixed!

File: libs/pages/onboarding/feature/src/lib/components/form-company/form-company.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
+import { Dispatch, SetStateAction, useEffect } from 'react'
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router'
+import { useUser } from '@console/domains/user'
+import { StepCompany } from '@console/pages/onboarding/ui'
+import { ONBOARDING_MORE_URL, ONBOARDING_URL } from '@console/shared/utils'
+
+const dataSize = [
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/pages/onboarding/feature/src/lib/components/form-company/form-company.tsx]: fixed!

**Comment by RemiBonnet** [comment]
- Adding "Onboarding" application with the right behavior

**Pull request merged**